Library/Formula/yodl.rb

```diff
@@ -0,0 +1,50 @@
+require 'formula'
+
+class Yodl < Formula
+  url 'http://downloads.sourceforge.net/project/yodl/yodl/3.05.01/yodl_3.05.01.orig.tar.gz'
+  homepage 'http://yodl.sourceforge.net/'
+  sha1 '94d8e59a8569a9d6fb4dc7a1b8e7d430f018e570'
+
+  depends_on "ghostscript"
+  depends_on "icmake"
+
+  patch :DATA
```

Comment:
Please add an explanation of this patch & link to the upstream bug/submission, Thanks.